crates/sui-core/src/unit_tests/move_integration_tests.rs

```diff
@@ -160,9 +160,10 @@ async fn test_object_wrapping_unwrapping() {
         (
             effects.created.len(),
             effects.deleted.len(),
+            effects.unwrapped_then_deleted.len(),
             effects.wrapped.len()
         ),
-        (1, 0, 1)
+        (1, 0, 0, 1)
```

Comment:
@tnowacki see here for confirmation that wraps don't add to `effects.deleted`

Reply:
(Just for anyone like me who was momentarily confused here: There's a separate `wrapped` effect that wraps go into which is what the last the number in this tuple represents, and there's a general principle that these lists in effects should be pairwise disjoint).